README.md

```diff
@@ -10,6 +10,35 @@ To work with containers, this package provides the helper script `setup_workspac
 
 Run `setup_workspace.sh --help` for more details about the script usage.
 
+If you're using [ORise container](https://github.com/orise-robotics/ros_ws), it assumes that the folder `/home/orise` will be used as your woskspace to develop using ROS 2 and also, your VSCode workspace. If you want to use different workspaces inside the container, just copy the files `.vscode-format` and `ros2.code-workspace` to the root of your ROS workspace and redefine the `COLCON_WORKSPACE_FOLDER` variable. For example, let's assume that you want to use a ROS 2 woskpace called *foxy_ws*, this folder should have:
```

Comment:
Markdownlint complains about this line size. Consider splitting them into two lines (a single enter do not jump lines in the rendered output)